cmd/wio/toolchain/npm/resolve/install.go

```diff
@@ -29,7 +31,7 @@ func (i *Info) InstallResolved() error {
 }
 
 func (i *Info) install(name string, ver string, data *npm.Version) error {
-    local, err := tryFindConfig(name, ver, i.dir)
+    local, err := tryFindConfig(&Node{Name: name, ResolvedVersion: semver.Parse(ver)}, i.dir)
```

Comment:
I know I said to pass stuff around using structs but it's kinda pointless if you just pack them into a struct and then unpack them later lol